docs/graphs/graph-examples/uniquevertices-examples.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 100
+title: uniqueVertices Examples
+---
+
+This page provides examples of traversing graphs using the `uniqueVertices` option. For more information about this option, refer to [Traversal Query Syntax](../graph-queries/traversal-queries/traversal-syntax.md).
+
+## Use Case: Identify Unique Viewers for Personalized Recommendations
+
+Consider an OTT video streaming service that wants to improve its content recommendations for viewers. The service wants to identify unique viewers who have recently watched content from a specific genre in order to offer them personalized recommendations and enhance their streaming experience. This approach can help the streaming service increase viewer engagement and promote content tailored to individual preferences.
+
+### Unique Viewers Graph Structure
+
+The streaming platform has the following graph structure:
+
+- Vertex collections: `users`, `content`
+- Edge collections: `watched` (edges represent the "watched" relationship between users and content)
+
+### C8QL Query
+
+The `uniqueVertices` option can be used in a [C8QL](../../queries/c8ql/) query to find unique viewers who have recently watched content from the 'Sci-Fi' genre:
+
+```sql
+FOR user, edge, path IN 1..1 OUTBOUND 'content/Sci-Fi' watched
+  OPTIONS {uniqueVertices: 'global'}
+  RETURN user
+```
```

Comment:
maybe something to add :
“global” – it is guaranteed that each vertex is visited at most once during the traversal, no matter how many paths lead from the start vertex to this one. If you start with a min depth > 1 a vertex that was found before min depth might not be returned at all (it still might be part of a path). Note: Using this configuration the result is not deterministic any more. If there are multiple paths from startVertex to vertex, one of those is picked. It is required to set bfs: true because with depth-first search the results would be unpredictable.

Reply:
focus is on BFS

Reply:
What do you mean by "focus is on BFS"?

Reply:
@diana-macrometa 
I didn't write it well, I was thinking that is good to add the following info:
Note: Using this configuration the result is not deterministic anymore. If there are multiple paths from startVertex to vertex, one of those is picked. It is required to set bfs: true because with depth-first search the results would be unpredictable.

Reply:
This topic is an example, not the syntax documentation. The syntax documentation is linked in the page intro and is ALMOST exactly the same as what you have written here. I updated it with the missing bit, please take a look.